shared/js/background/message-handlers.js

```diff
@@ -345,6 +345,7 @@ const messageHandlers = {
     getBrowser,
     openOptions,
     submitBrokenSiteReport,
+    getBreakageFormOptions: getDisclosureDetails,
```

Comment:
_(I normally would avoid adding message handlers here, but I got in a circular dependency mess with the unit tests and didn't have time to debug that. Just adding the handler here was easier.)_